src/components/accordion/accordionReducer.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
```

Comment:
Can we move this reducer to accordion component itself? It's much easier to follow when such code is close to the component (to keep changes in sync). We can argue whether we also should move AccordionItem to the same file and it would be correct thinking as well